OpenTrons/opentrons-api
# feature(app, odd): show full error list in the run summery

# Overview

closes https://opentrons.atlassian.net/browse/EXEC-635 (minos a few bug fixes ;-) ) 
show full error list in the run summery modal. 

## Test Plan and Hands on Testing

- run the desktop app/odd.
- enter ER mode and make sure you encountered command errors. 
- finish the run and click the view error details -> make sure you can see the error encountered. 

## Changelog

- added get calls for the run command error list
- updated FailedRunModal to show the full error list if we got it. 

## Risk assessment

low. should not affect existing code.
